behind-the-veil-siteroot/imports/ui/components/pages/artist/ArtistServiceArea.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * File Description: Service Area for artists within the settings page
+ * File version: 1.0
+ * Contributors: Hirun
+ */
+import React from "react";
+import WhiteBackground from "../../whiteBackground/WhiteBackground.jsx";
+import PageLayout from "../../../enums/PageLayout";
+import Tabs from "../../tabs/Tabs.jsx";
+import Button from "../../button/Button.jsx";
+import { ArrowLeftIcon, ArrowRightIcon } from '@heroicons/react/24/outline'
+
+/**
+ * This page allows the artist to enter to enter a location as well as provide a radius in which they can travel from that location
+ */
+
+export const ArtistServiceArea = () => {
+  const ServiceAreaPanel = (
+    <div className="grid grid-cols-12 gap-4">
+      <div className="col-span-4 ml-9 large-text">Location</div>
+      <div className="col-span-1 text-center large-text">Radius (Km)</div>
+      <div className="col-span-7"></div> {/* Empty div for spacing */}
+      <div className="col-span-4 ml-9 main-text text-dark-grey">
+        Melbourne CBD
+      </div>
+      <div className="col-span-1 text-center main-text text-dark-grey">15</div>
+      <div className="col-span-5 text-right">
+        {/* the delete icon would go here */}
+      </div>
+      <div className="col-span-4 ml-9 main-text text-dark-grey">
+        Melbourne Airport
+      </div>
+      <div className="col-span-1 text-center main-text text-dark-grey">3</div>
+      <div className="col-span-7 text-right">
+        {/* the delete icon would go here */}
+      </div>
+      <div className="col-span-4 ml-9">
+        <input
+          type="text"
+          style={{
```

Comment:
instead of using regular inline styles, try to use tailwind's className system, i.e. something like
`className="rounded outline outline-1 outline-light-grey p-2"`